jmix-dynattr/dynattr/src/main/java/io/jmix/dynattr/model/CategoryAttribute.java

```diff
@@ -52,8 +52,10 @@
 public class CategoryAttribute implements Serializable {
     private static final long serialVersionUID = -6959392628534815752L;
 
+    static final String CODE_FIELD_REGEXP = "^[a-z]\\w*$";
+
     @Id
-    @Column(name = "ID")
+    @Column(name = "ID", nullable = false)
```

Comment:
Get rid of unrelated changes

Reply:
It is unwilled mistake. I haven't seen all changes just before commit.